mitodl/mitxonline
# Update product description help text in CMS

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #179 

#### What's this PR do?
- Updates the help text for the description field in the product/course model

#### How should this be manually tested?
- Visit any existing course page or create a new one under the courses page
- Hover over the description field and you should see an updated description message (See screenshot for ref) 


#### Screenshots (if appropriate)
<img width="1262" alt="Screenshot 2021-09-17 at 5 25 42 PM" src="https://user-images.githubusercontent.com/34372316/133783054-90efa650-2a23-4fe5-905f-ec05f83a769d.png">
